.jscsrc

```diff
@@ -50,6 +50,8 @@
 		"examples/typescript-*/js/**/*.js",
 		"examples/vanilladart/**/*.js",
 		"examples/vanilla-es6/dist/bundle.js",
+		"examples/react-redux/js/reducers/*.js",
```

Comment:
Check http://stackoverflow.com/questions/25223149/is-there-any-way-for-jscs-to-ignore-rules-per-file-block-or-line

Quoting: 
To ignore all rules

// Code here will be linted with JSCS.
// jscs:disable
// Code here will be ignored by JSCS.
// jscs:enable
To ignore a specific rule:

// Code here will be linted with JSCS.
// jscs:disable specificRule
// Code here will be ignored by JSCS.
// jscs:enable specificRule